src/app.rs

```diff
@@ -396,11 +380,7 @@ where
         // case) causes a directory to be renamed and another extracted. We only want to execute
         // this if we need to (if `api.public()` is `None`) so we disable the lint.
         #[allow(clippy::unnecessary_lazy_evaluations)]
-        server
-            .at("/public")
-            .at(&format!("v{version}"))
-            .at(&prefix.join("/"))
-            .serve_dir(api.public().unwrap_or_else(|| &DEFAULT_PUBLIC_PATH))?;
+        server.at(&format!("v{version}")).at(&prefix.join("/"));
```

Comment:
What does this do now if we're not serving a directory or adding any endpoints after `at(...)`?

Also, the comment and `#[allow]` attribute above are no longer needed